### PR TITLE
Add Studio Kestra Nue PCB

### PR DESCRIPTION
**Pull request by studiokestra**
## Description

Add QMK and VIA support for Nue 60% PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/studiokestra/nue/config.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0727
```
Comment (drashna) [inline keyboards/studiokestra/nue/config.h:23]: This vendor ID is already in use, and needs to be changed. 
https://www.the-sz.com/products/usbid/index.php?v=0x0727&p=&n=

File: keyboards/studiokestra/nue/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 Studio Kestra
```
Comment (drashna) [inline keyboards/studiokestra/nue/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/studiokestra/nue/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/* Copyright 2021 Studio Kestra
```
Comment (drashna) [inline keyboards/studiokestra/nue/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/studiokestra/nue/nue.c [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "nue.h"
+
+void matrix_init_kb(void) {
+	
+	matrix_init_user();
+}
```
Comment (drashna) [inline keyboards/studiokestra/nue/nue.c:22]: This is the default behavior, and can be removed. 
```suggestion
```

File: keyboards/studiokestra/nue/readme.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Nue
```
Comment (drashna) [inline keyboards/studiokestra/nue/readme.md:1]: Could you add instructions on how to jump to the bootloader, so the firmware can be flashed?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/studiokestra/nue/config.h [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7C10
```
Comment (drashna) [inline keyboards/studiokestra/nue/config.h:23]: 👍🏻

**Review by drashna — APPROVED** [review]
LGTM

**Review by fauxpark — APPROVED** [review]

**Pull request merged**